conda/recipes/rmm/recipe.yaml

```diff
@@ -76,6 +76,7 @@ requirements:
     from_package:
       - ${{ compiler("cuda") }}
       - cuda-python
+      - python  # required to avoid an incorrect runtime python_abi requirement from `host`
```

Comment:
Incorrect how? I would expect that to be correct.

Reply:
`python_abi       ┆ 3.12.* *_cp312 (RE of [host: python])     `  for a python 310 build, for one example

Reply:
This might be the wrong fix

Reply:
Might need to constrain `host` to `py_version` instead

Reply:
It is not clear to me how this was working before